missas/core/templates/home.html

```diff
@@ -0,0 +1,215 @@
+                            Projeto Open Source
+                        </h4>
+                        <p class="mb-2">
+                            O Missas.com.br é um projeto <strong>100% gratuito e de código aberto</strong>.
+                            Acreditamos na transparência e na colaboração da comunidade para tornar esta ferramenta cada vez melhor.
+                        </p>
+                        <p class="mb-0">
+                            <i class="fa-solid fa-code me-1"></i>
+                            Contribua conosco, reporte bugs ou sugira melhorias!
+                        </p>
+                    </div>
+                    <div class="col-12 col-md-4 text-center mt-3 mt-md-0">
+                        <a href="https://github.com/lucasrcezimbra/missas.com.br"
+                           target="_blank"
+                           rel="noopener noreferrer"
+                           class="btn btn-success btn-lg">
+                            <i class="fa-brands fa-github me-2"></i>
+                            Ver no GitHub
+                        </a>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <!-- Contato e Contribuição -->
+    <div class="row">
+        <div class="col-12 col-md-6 mb-3">
+            <div class="card h-100 border-0 shadow-sm"
+                 style="background: linear-gradient(135deg, #E8F5E8, #C8E6C9)">
+                <div class="card-body text-center">
+                    <i class="fa-brands fa-whatsapp"
+                       style="font-size: 3rem;
+                              color: #25D366;
+                              margin-bottom: 1rem"></i>
+                    <h5 class="card-title" style="color: #2E7D32; font-weight: 600;">Fale Conosco</h5>
+                    <p class="card-text mb-3">Tem alguma dúvida ou sugestão? Entre em contato pelo WhatsApp!</p>
+                    <a href="https://wa.me/+5584991234449"
+                       target="_blank"
+                       rel="noopener noreferrer"
+                       class="btn btn-success">
+                        <i class="fa-brands fa-whatsapp me-2"></i>
+                        Enviar Mensagem
+                    </a>
+                </div>
+            </div>
+        </div>
+        <div class="col-12 col-md-6 mb-3">
+            <div class="card h-100 border-0 shadow-sm"
+                 style="background: linear-gradient(135deg, #E3F2FD, #BBDEFB)">
+                <div class="card-body text-center">
+                    <i class="fa-solid fa-plus-circle"
+                       style="font-size: 3rem;
+                              color: var(--primary-blue);
+                              margin-bottom: 1rem"></i>
+                    <h5 class="card-title"
+                        style="color: var(--primary-blue);
+                               font-weight: 600">Adicione uma Paróquia</h5>
```

Comment:
Move this higher in the page. Before the stats. Add remove the modal. It should be the form directly in the home page.